CMP1903_A1_2324/Game.cs

```diff
@@ -15,7 +17,57 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
+        //Objects
+        Die d1 = new Die();
+        Die d2 = new Die();
+        Die d3 = new Die();
+        private int Roll1;
+        private int Roll2;
+        private int Roll3;
+        private int CurrentTotal;
+        private int Numof3InaRow = 0;
+
         //Methods
 
+        //Rolls all 3 die, with a 1ms pause in between rolls to allow for possibility of non-duplicates to occur
+        public void RollDie()
+        {
+            Roll1 = d1.Roll(); 
+            Thread.Sleep(1); 
+            Roll2 = d2.Roll(); 
+            Thread.Sleep(1);
+            Roll3 = d3.Roll();
+            Thread.Sleep(1);
+            if (Roll1 == Roll2 && Roll2 == Roll3)
+            {
+                Numof3InaRow += 1; //If all rolls are the same, increases the 3 in a row count by 1
+            }
+        }
+        
+        //Returns rolls
+        public int GetRoll1()
+        {
+            return Roll1;
+        }
+        public int GetRoll2()
+        {
+            return Roll2;
+        }
+        public int GetRoll3()
+        {
+            return Roll3;
+        }
+
+        //Returns total
+        public int GetTotal()
+        {
+            return (Roll1 + Roll2 + Roll3);
+        }
+
```

Comment:
Instead of creating 3 Roll methods to return 3 values you could have one method that returns all 3 values in a tuple or array.